code/modules/research/designs/autolathe_designs.dm

```diff
@@ -290,14 +290,32 @@
 	category = list("hacked","Misc","Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_CARGO
 
-/datum/design/generic_gas_tank
-	name = "Generic Gas Tank"
-	id = "generic_tank"
+/datum/design/generic_gas_tank_t1
+	name = "Generic Tier 1 Gas Tank"
+	id = "generic_tank_t1"
 	build_type = AUTOLATHE | PROTOLATHE
 	materials = list(/datum/material/iron = 1000)
-	build_path = /obj/item/tank/internals/generic
+	build_path = /obj/item/tank/tier_1
 	category = list("initial","Misc","Equipment")
-	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_CARGO
+	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_CARGO | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/generic_gas_tank_t2
+	name = "Generic Tier 2 Gas Tank"
+	id = "generic_tank_t2"
+	build_type = AUTOLATHE | PROTOLATHE
+	materials = list(/datum/material/iron = 1000, /datum/material/plasma = 1000)
+	build_path = /obj/item/tank/tier_2
+	category = list("Misc","Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_CARGO | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/generic_gas_tank_t3
+	name = "Generic Tier 3 Gas Tank"
+	id = "generic_tank_t3"
+	build_type = AUTOLATHE | PROTOLATHE
```

Comment:
wait, if you can just autolathe these, what's the point of locking them behind a tech node?

Reply:
I'll fix that